developer/src/kmc-kmn/test/test-messages.ts

```diff
@@ -80,4 +80,11 @@ describe('CompilerMessages', function () {
     assert.equal(callbacks.messages[0].message, "A key on layer \"default\" has no identifier.");
   });
 
+  // ERROR_NotSupportedInKeymanWebOutput
+
+  it('should generate ERROR_NotSupportedInKeymanWebOutput if a rule has `return` in the output', async function() {
+    await testForMessage(this, ['invalid-keyboards', 'error_not_supported_in_keyman_web_output.kmn'], KmnCompilerMessages.ERROR_NotSupportedInKeymanWebOutput);
+    assert.equal(callbacks.messages[0].message, "Statement 'return' is not currently supported in output for web and touch targets");
+  });
```

Comment:
Can we assert line 11 in the kmn is also reported in the compiler message?

Reply:
See topmost comment:

>Potential future improvements to unit tests:
>
>Tests for the other messages listed here
>Tests for line number correctness